### DISCUSSION TITLE
Feature Request: Command that completely clears out the database

### DISCUSSION THREAD

**Discussion by jovelluna12** [laravel/framework#52022] [Ideas] [answered]
Currently, the migrate command works by running each migration file alphabetically, This causes issues like [this](https://github.com/laravel/framework/issues/8002). While it was explained in the issue linked that the migrations is supposed to be this way due to the way migrations and databases fundamentally works, I encountered a case where I need to clear out the entire database of the values from the different models that a project im working on currrently have. and seeing that there is currently no command or feature that does this, the migrate reset or migrate refresh command is the closest thing that I can think of, besides actually removing these values from the database itself, which I ended up doing anyway.

**Comment by crynobone** [comment, marked as answer]
```bash
php artisan db:wipe
```

**Reply by jovelluna12** [reply]
> ```shell
> php artisan db:wipe
> ```

oh woow...  i guess i didn't look properly :/
